src/initStarter.js

```diff
@@ -146,6 +161,73 @@ const writeReadme = async (rootPath, name, type) => {
   }
 };
 
+const replaceInFilesByType = async (patterns, template, newValue) => {
+  const options = {
+    files: patterns,
+    from: new RegExp(template, 'g'),
+    to: newValue,
+  };
+  try {
+    const res = await replaceInFile(options);
+    return res.filter((f) => f.hasChanged);
+  } catch (e) {
+    return [];
+  }
+};
+
+const replaceProjectName = async (rootPath, fullName, projectName) => {
+  const prevDir = process.cwd();
+  console.log('branding project...');
+  process.chdir(rootPath);
+  try {
+    // replace in yaml
+    const yamlFiles = await Promise.all(
+      ['.github/workflows/*.yaml', '.github/workflows/*.yml'].map(
+        async (pattern) =>
+          replaceInFilesByType(pattern, YAML_TEMPLATE, fullName)
+      )
+    );
+    console.log(
+      'Yaml files:',
+      yamlFiles.flat().map((f) => f.file)
+    );
+
+    // replace in html
+    const htmlFiles = await replaceInFilesByType(
+      'public/*.html',
+      HTML_TEMPLATE,
+      fullName
+    );
+    console.log(
+      'HTML files:',
+      htmlFiles.map((f) => f.file)
+    );
+
+    // replace name in package.json
+    replaceJSON('./package.json', 'name', projectName, { silent: true });
+    replaceJSON('./public/manifest.json', 'name', projectName, {
+      silent: true,
+    });
+    replaceJSON('./public/manifest.json', 'short_name', projectName, {
+      silent: true,
+    });
+  } catch (e) {
+    console.error(e);
```

Comment:
@spaenleh, in these errors is where we need to add the logger / parser to ensure that messages are bubbled up. Maybe for another PR.